### ISSUE TITLE
[Accessibility][Keyboard -Create Node]: Status message for the node creation is not announced by the narrator.

### ISSUE THREAD

**Issue by ashish315** [microsoft/BotFramework-Composer#2079]
**User Experience** 
The AT user might gets confused as no confirmation message is announced for the node creation. If the status messages are not narrated by the narrator then the screen reader dependent user will not be able to know the confirmation of the actions performed by him.

**Test Environment**
Browser: Microsoft Edge Canary Version: 82.0.425.0 (Official build)
OS build: 2004 (19564.1005)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)

**Repro Steps:** 
1. Open the above mentioned URL in EDGE and set the zoom at 400%.
2. Select 'Design Flow' from left navigation.
3. Now, navigate to the flow chart.
4. Navigate to "+" button and activate it.
5. Create a node by selecting any of the available.
6. Observe the issue.

**Actual Result:**
Screen reader doesn’t provide any confirmation on creating 'New node'. New node gets added but screen reader doesn't provide any confirmation.

**Expected:**
The narrator should announce any status message such as it should announce as "New node is created".

**Note:**
Same issue observed throughout the "Design Flow" page such as copy, cut, paste, node creation, Delete, New trigger creation, new dialog creation etc.

**MAS Impacted:** [MAS4.1.3](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/Doc.aspx?sourcedoc=%7B684573A7-B089-4131-9B39-0009054125D3%7D&file=MAS%204.1.3%20%E2%80%93%20Status%20Messages.docx&action=default&mobileredirect=true&cid=6a538745-b402-41f3-85ea-9f67151fa4c9)

**Attachment:**
[MAS4.1.3_Status messages are not announced.zip](https://github.com/microsoft/BotFramework-Composer/files/4249592/MAS4.1.3_Status.messages.are.not.announced.zip)

**Comment by liweitian** [comment]
We have changed the UI. The issue does not exist anymore.

**Comment by yeze322** [comment]
Solution: Need a hook after node creation to trigger Screen Reader.
To be investigated: how to trigger Screen Reader?
To do: if it's a required function, we should also apply the behavior to 'New Dialog...' 'New Trigger...'

**Comment by corinagum** [comment]
@yeze322 you can look into having an `aria-live` region that will send updates to AT when there are changes to the child DOM. Feel free to ping me if you have questions

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions

**Comment by cwhitten** [comment]
This can also be addressed by hooking into @beyackle `live-region` in #2050 . 

@yeze322 @alanlong9278 can you figure out an appropriate event mechanism that @beyackle can hook into to send this announcement? see https://github.com/microsoft/BotFramework-Composer/issues/2312

**Comment by yeze322** [comment]
@cwhitten I, Andy, Dong are in the loop of 2321, we will figure out an event mechanism to support both ScreenReader and plugin system

**Comment by alanlong9278** [comment]
Add screen reader in visual editor. 
```
export enum ScreenReaderMessage {
  EventFocused = 'Event focused',
  NodeFocused = 'Node focused',
  DialogOpened = 'Dialog opened',
  NodeDeleted = 'Node deleted',
  NodesDeleted = 'Nodes deleted',
  NodeCreated = 'Node created',
  NodesCreated = 'Nodes created',
  EventCreated = 'Event created',
  NodesCopied = 'Nodes copied',
  NodesCut = 'Nodes cut',
  NodesMoved = 'Nodes moved',
  ActionUndo = 'Action undo',
  ActionRedo = 'Action redo',
}
```

**Comment by ashish315** [comment]
As checked issue is resolved So HCL is closing the issue.
